### PR TITLE
fix: home responsive buttons

### PR DESCRIPTION
**Pull request by fersilva16**
# Before
image

# After
image

### CODE REVIEW

File: apps/web/src/components/ActionButtons.tsx [hunk 1]
```diff
@@ -40,6 +41,7 @@ export const ActionButtons = () => {
           colorScheme="purple"
           leftIcon={<SiSubstack />}
           mt={{ base: '0', md: '10px' }}
+          width="100%"
```
Comment (sibelius) [inline apps/web/src/components/ActionButtons.tsx:44]: Flex 1?

**Review by sibelius — APPROVED** [review]

File: apps/web/src/components/ActionButtons.tsx (hunk 1 above)
Comment (fersilva16) [reply apps/web/src/components/ActionButtons.tsx:44]: Makes sense, will fix it!

Comment (fersilva16) [reply apps/web/src/components/ActionButtons.tsx]: Maybe not
image

Comment (noghartt) [reply apps/web/src/components/ActionButtons.tsx:44]: I think that you can do something like it:

```jsx
// ...
<VStack>
  // Tweet button
  <HStack>
    // Social Buttons here
  </HStack>
  // Other buttons
</VStack>

Comment (fersilva16) [reply apps/web/src/components/ActionButtons.tsx:44]: `width: 100%` seems better to me

**Pull request merged**